### PR TITLE
Lausanne multiscale atlases

### PR DESCRIPTION
**Pull request by frheault**
Improved docs, new atlases, match with the singularity on Braindata AND Beluga

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # FreeSurfer pipeline
 ===================
 
-Run the FreeSurfer recon-all pipeline.
+Run the FreeSurfer recon-all pipeline and create the customize freesurfer,
+brainnetome and glasser connectivity atlas in native space.
```
Comment (mdesco) [inline README.md:5]: freesurfer, Lausanne scale 1-5, brainnetome ...

File: USAGE [hunk 2]
```diff
@@ -1,7 +1,22 @@
 FreeSurfer pipeline
 ===================
 
-Run recon-all on all encountered subjects
+Run recon-all on all subjects and create the customize freesurfer,
+brainnetome and glasser connectivity atlas in native space. This pipeline also
+creates the 5 scales of the 2008 Lausanne Atlas. It is also possible to run
+this nextflow if your FreeSurfer is already computed.
```
Comment (mdesco) [inline USAGE:7]: Can you add refs to the USAGE please.
Lausanne has a citation in the Readme.
Brainnetome paper
Glasser Nature paper

**Review by mdesco — COMMENTED** [review]
I will need the new singularity to test this PR

**Comment by frheault** [comment]
@mdesco should work with freesurfer_14072020.img in ~/projects/rrg-descotea/containers/

**Comment by mdesco** [comment]
No. I will fetch the error...

**Comment by mdesco** [comment]
I get this error and I assumed that things were missing in the singularity image...
executor >  ignite (30)
[db/647bdc] process > Generate_Atlases_Lausanne [120%] 30 of 25, failed: 30
[82/6f72d4] process > Generate_Atlases_FS_BN_GL [100%] 5 of 5, failed: 5
WARN: Killing pending tasks (27)
ERROR ~ Error executing process > 'Generate_Atlases_Lausanne (jeremie-01)'

Caused by:
  Process `Generate_Atlases_Lausanne (jeremie-01)` terminated with an error exit status (2)

Command executed:

  ln -s /FS_BN_GL_utils//fsaverage $(dirname jeremie-01)/
  freesurfer_home=$(dirname $(dirname $(which mri_label2vol)))
  python3 /FS_BN_GL_utils//lausanne_multi_scale_atlas/generate_multiscale_parcellation.py $(dirname jeremie-01) jeremie-01 $freesurfer_home --scale 3 --dilation_factor 0 --log_level DEBUG

  mrthreshold jeremie-01/mri/rawavg.mgz mask.nii.gz -abs 0.001 -quiet
  scil_reshape_to_reference.py jeremie-01/mri/lausanne2008.scale3+aseg.nii.gz mask.nii.gz lausanne_2008_scale_3.nii.gz --interpolation nearest
  scil_image_math.py convert lausanne_2008_scale_3.nii.gz lausanne_2008_scale_3.nii.gz --data_type int16 -f
  scil_dilate_labels.py lausanne_2008_scale_3.nii.gz lausanne_2008_scale_3_dilate.nii.gz --distance 2 --mask mask.nii.gz

  cp /FS_BN_GL_utils//lausanne_multi_scale_atlas/*.txt ./
  cp /FS_BN_GL_utils//lausanne_multi_scale_atlas/*.json ./

Command exit status:
  2

Command output:
  (empty)

Command error:
  python3: can't open file '/FS_BN_GL_utils//lausanne_multi_scale_atlas/generate_multiscale_parcellation.py': [Errno 2] No such file or directory

Work dir:
  /lustre04/scratch/descotea/Neurofeedback/output_brainnetome/work/ce/f82e70fb33225a18c5290f6435c215

Tip: when you have fixed the problem you can continue the execution appending to the nextflow command line the option `-resume`

 -- Check '.nextflow.log' file for details

**Review by mdesco — APPROVED** [review]
All good!

**Review by mdesco — COMMENTED** [review]
Is this PR ready to be merged or needs further work @frheault 

Can you change --root options to --input options as we discussed for other flows.

Thanks

**Pull request merged**